tket/src/Ops/include/Ops/ClassicalOps.hpp

```diff
@@ -30,7 +30,7 @@ namespace tket {
 class ClassicalOp : public Op {
  public:
   /**
-   * Construct a ClassicalOp of specified shape
+   * Construct a ClassicalEvalOp of specified shape
```

Comment:
this change is wrong right?

please document in this file also the difference between ClassicalOp and ClassicalEvalOp